index.js

```diff
@@ -179,6 +186,14 @@ module.exports = async function(options) {
 
   self.dashboard = await spinUpDashboard();
 
+  if (options.maxRequestsBeforeShutdown) {
```

Comment:
I was going to suggest testing that this is a number, but looking below it won't crash if this is `NaN` in the middleware. Any reason not to include a warning if it isn't, though?

Reply:
Currently we don't type-check options anywhere in Apostrophe. I don't want to open that box just for this one. We could introduce rigor for options in a future release, and that would be good, although it is already getting late to do it in 3.x.